src/cli.ts

```diff
@@ -22,6 +22,7 @@ import * as os from 'os';
 import * as fs from 'fs';
 import * as playwright from 'playwright';
 import { Browser, BrowserContext, Page } from 'playwright';
+import { electron } from 'playwright-electron'
```

Comment:
We can do

`(playwright as any).electron`

and make `electron` itself a dev dependency, that way it only works for those that have electron installed.

Reply:
Thanks, however I tried this, but it looks like `(playwright as any).electron` is undefined. The only keys on playwright are:
```
_events
_eventsCount
_maxListeners
_objects
_connection
_type
_guid
_parent
_logger
_channel
_initializer
chromium
firefox
webkit
_clank
devices
selectors
errors
_toImpl
```

Reply:
Yes, just looked at that code myself, saw that I was lying to you and came here to see this comment. I'll change it though to be there upstream.